### ISSUE TITLE
Do not announce "cursor added" if not cursor is added

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#118965]
1. Have only one empty line of text
2.  add secondary cursor below
3. Screen reader announces "Cursor Added"

While in fact no cursor was added 🐛
